static/app/components/searchSyntax/parser.tsx

```diff
@@ -600,6 +607,12 @@ export class TokenConverter {
       return this.checkInvalidInFilter(value as InFilter['value']);
     }
 
+    if (filter === FilterType.AggregateGeneric) {
```

Comment:
Check generic aggregate values as text to get an error message if it is submitted with no value.

Or is it better to introduce a separate method `checkForEmptyValue`?